en_us/en_us.rc

```diff
@@ -507,22 +507,23 @@ BEGIN
     PUSHBUTTON      "&Copy",ID_EDIT_COPY,300,76,50,14
 END
 
-IDD_LOGIN DIALOGEX 0, 0, 210, 97
+IDD_LOGIN DIALOGEX 0, 0, 230, 98
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 CAPTION "ECL IDE"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     LTEXT           "&Configuration:",IDC_STATIC_CONFIGURATION,7,9,51,8
-    COMBOBOX        IDC_COMBO_CONFIG,61,7,141,65,CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX        IDC_COMBO_CONFIG,61,7,161,65,CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP
     LTEXT           "&Login ID:",IDC_STATIC_USER,7,26,34,8
-    EDITTEXT        IDC_EDIT_USER,61,23,142,14,ES_AUTOHSCROLL
+    EDITTEXT        IDC_EDIT_USER,61,23,162,14,ES_AUTOHSCROLL
     LTEXT           "&Password:",IDC_STATIC_PASSWORD,7,43,38,8
-    EDITTEXT        IDC_EDIT_PASSWORD,61,40,142,14,ES_PASSWORD | ES_AUTOHSCROLL
-    CTEXT           "",IDC_STATIC_MSG,7,58,195,16
-    PUSHBUTTON      "Pre&ferences...",IDC_LOGIN_PREFS,7,78,54,14,WS_DISABLED
-    DEFPUSHBUTTON   "OK",IDOK,101,78,50,14
-    PUSHBUTTON      "Cancel",IDCANCEL,153,78,50,14
-    CTEXT           "",IDC_STATIC_UPGRADE,7,58,195,16
+    EDITTEXT        IDC_EDIT_PASSWORD,61,40,162,14,ES_PASSWORD | ES_AUTOHSCROLL
+    CTEXT           "",IDC_STATIC_MSG,64,58,154,16
+    PUSHBUTTON      "Pre&ferences",IDC_LOGIN_PREFS,4,78,55,14,WS_DISABLED
+    DEFPUSHBUTTON   "OK",IDOK,117,78,50,14
+    PUSHBUTTON      "Cancel",IDCANCEL,172,78,50,14
+    CTEXT           "",IDC_STATIC_UPGRADE,61,58,141,16
+    PUSHBUTTON      "Change Psswd",IDC_BUTTON_CHANGEPASSWORD,4,61,54,14
```

Comment:
Screenshot?

Reply:
![amtlogin01](https://user-images.githubusercontent.com/10133951/51198394-22518280-18c2-11e9-9fdc-48c4b965e699.PNG)

Reply:
![login01](https://user-images.githubusercontent.com/10133951/51198978-a9532a80-18c3-11e9-996b-5d8d26164db5.PNG)

Reply:
@GordonSmith made this longer. Will combine the other jira as a subtask which people were complaining about the small size of the configuration pulldown.